src/main.cpp

```diff
@@ -269,12 +272,14 @@ void setup()
     ESP_LOGI(LOG_TAG, "Target: %s", PIO_ENV);
     ESP_LOGI(LOG_TAG, "Version: %s", VERSION);
     ESP_LOGI(LOG_TAG, "Hostname: %s", gSettingsHostname.c_str());
+    ESP_LOGI(LOG_TAG, "Partition: Factory");
 
     /* Start wifi */
     (void)WiFi.mode(WIFI_STA);
 
     setupOta();
     setupWebServer();
+    setAppPartition0Active();

```

Comment:
Its ok for now, later we need to check whether on the app partition is something senseful.